app/assets/javascripts/service_banner.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener("turbolinks:load", function() {
+    var buttons = document.getElementsByClassName('close-service-notification');
+
+    for (var i = 0; i < buttons.length; i++) {
+        if (localStorage.getItem(buttons[i].id)) {
```

Comment:
It might be a good idea to prefix the notification IDs so they're namespaced. Whilst we don't use JavaScript `localStorage` for anything else right now, we don't want to pollute the global namespace with notification IDs and then later find that those IDs clash with other JS features we want to release.

So for example, given a service notification with ID `"early-allocations-released"`, we could use a `localStorage` key of something like `"ServiceNotifications.early-allocations-released"`.